Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3995,6 +3995,20 @@ bool CSSPropertyAnimation::isPropertyAdditiveOrCumulative(AnimatableProperty pro
     );
 }
 
+static bool syntaxValuesRequireBlendingForAccumulativeIteration(const CSSCustomPropertyValue::SyntaxValue& a, const CSSCustomPropertyValue::SyntaxValue& b, bool isList)
+{
+    return WTF::switchOn(a, [b, isList](const Length& aLength) {
+        ASSERT(std::holds_alternative<Length>(b));
+        return isList ? false : lengthsRequireBlendingForAccumulativeIteration(aLength, std::get<Length>(b) );
```

Comment:
could be

`!isList && lengthsRequireBlendingForAccumulativeIteration(aLength, std::get<Length>(b) );`